frontend/package.json

```diff
@@ -9,23 +9,33 @@
     "lint": "yarn next lint"
   },
   "dependencies": {
+    "@svgr/webpack": "^7.0.0",
     "@types/node": "18.15.0",
     "@types/react-dom": "18.0.11",
     "@types/react-router": "^5.1.20",
+    "daisyui": "^2.51.5",
     "eslint": "8.36.0",
     "eslint-config-next": "13.2.4",
     "next": "13.2.4",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "typescript": "4.9.5"
+    "react-router-dom": "^6.9.0",
+    "react-smart-flexbox": "^0.0.7",
+    "styled-components": "^5.3.9",
+    "typescript": "4.9.5",
+    "webpack": "^5.76.3"
```

Comment:
Why are you guys using react-router in a NextJS project? You should not need this for routing. You should be consulting the documentation I provided to you guys.